src/ApiService/ApiService/onefuzzlib/ScalesetOperations.cs

```diff
@@ -5,6 +5,9 @@ namespace Microsoft.OneFuzz.Service;
 public interface IScalesetOperations : IOrm<Scaleset>
 {
     IAsyncEnumerable<Scaleset> Search();
+
+    public IAsyncEnumerable<Scaleset?> SearchByPool(string poolName);
```

Comment:
the type paramter should not be nullable 
```suggestion
    public IAsyncEnumerable<Scaleset> SearchByPool(string poolName);
```